src/main/controllers/SubscriptionConfigureListConfirmedController.ts

```diff
@@ -2,18 +2,39 @@ import { Response } from 'express';
 import { PipRequest } from '../models/request/PipRequest';
 import { SubscriptionService } from '../service/SubscriptionService';
 import { cloneDeep } from 'lodash';
+import { PendingSubscriptionsFromCache } from '../service/PendingSubscriptionsFromCache';
 
 const subscriptionService = new SubscriptionService();
 
 export default class SubscriptionConfigureListConfirmedController {
     public async post(req: PipRequest, res: Response): Promise<void> {
-        const result = await subscriptionService.configureListTypeForLocationSubscriptions(
-            req.user['userId'],
-            req.body['list-selections[]'],
-            []
-        );
+        const userId = req.user['userId'];
+        const cacheService = new PendingSubscriptionsFromCache();
 
-        if (result) {
+        if (req.body['list-language'] === undefined) {
```

Comment:
The check for language stuffs should go into the POST method in the list language controller. Need to update the nunjucks file to post to that route and not to this one.

Reply:
fixed.